xbmc/linux/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+file(GLOB VARIANTS_CPP ${CORE_SYSTEM_VARIANT}/*.cpp)
+file(GLOB VARIANTS_H ${CORE_SYSTEM_VARIANT}/*.h)
```

Comment:
globbing is never okay. i haven't thought much about this variant stuff, i thought it was all covered by the treedata construct.
if there are sub-variants needed for linux, we should add that there imo. ie a dedicated directory for variants and conditionally processing of those text files using something ala CORE_SYSTEM_VARIANT.